app/controllers/cases_controller.rb

```diff
@@ -104,7 +104,22 @@ def closed_cases
     respond_to do |format|
       format.html     { render :closed_cases }
       format.csv do
-        send_csv_cases'closed'
+        send_csv_cases 'closed'
+      end
+    end
+  end
+
+  # users only want to see cases deleted in the last 6 months
+  def deleted_cases
+    cases = Case::Base.unscoped
+              .soft_deleted
+              .updated_since(6.months.ago)
```

Comment:
Should 6.months be a constant?

Reply:
Not sure that a constant would make the code more readable here? It would be more code lines after possibly removing the comment